verification/curator-service/api/src/controllers/sources.ts

```diff
@@ -243,4 +250,39 @@ export default class SourcesController {
         }
         return;
     };
+
+    private async sendNotifications(
+        source: SourceDocument,
+        type: NotificationType,
+    ): Promise<void> {
+        if (source.notificationRecipients?.length > 0) {
```

Comment:
early return in case length === 0 ? just to keep indent left/positiveness whenever possible.

Reply:
SG, done.